src/pages/_app.tsx

```diff
@@ -29,6 +29,16 @@ declare global {
 export default class App extends NextApp {
   render() {
     const {Component, pageProps} = this.props
+    const AppComponent = Component as any
```

Comment:
casting this `as any` so that I can check to see if `getLayout` exists without a type error.